saleor/graphql/checkout/mutations.py

```diff
@@ -538,13 +599,19 @@ def perform_mutation(cls, _root, info, checkout_id, store_source):
                 )
 
         try:
-            billing_address = order_data["billing_address"]  # type: models.Address
-            shipping_address = order_data["shipping_address"]  # type: models.Address
+            billing_address = order_data["billing_address"]
```

Comment:
I don't think that this part of the code should be in `try`, `except`. Let's keep them as tight as possible 
`perform_mutation` is growing and growing, maybe we can extract this part to code to separate method.